app/services/scheduled_reporter.rb

```diff
@@ -0,0 +1,56 @@
+class ScheduledReporter
+  attr_reader :check_time
+
+  def initialize(check_time)
```

Comment:
interested in why you chose to pass in time rather than look at current time within this class - assume we will always be using current time and could mock time with Timecop in specs, but there are probably other reasons for doing this that I am not thinking of.